src/views/Questionnaire.vue

```diff
@@ -73,6 +73,7 @@ const previousQuestion = () => {
         aria-valuemin="1"
         :aria-valuemax="questionsCount"
         :aria-valuenow="currentQuestionProgress"
+        aria-hidden="true"
```

Comment:
Ah, I wanted to hide the text progress but set it to `false`, rather than `true`:

https://github.com/okfde/real-o-mat/blob/97cf72abae4a7e367fedab5fab2b79dc67206f8a/src/components/QuestionCard.vue#L65-L67

Should we rather hide the text instead of the progress bar (thought it would be the more "semantic" element).

Reply:
Hiding the text would be confusing for users with e.g. motion disabilities or screen reader users who use "explore by touch" on touchscreen devices, as it is generally a larger target area and its content is more meaningful. The progress bar is only a secondary visual element that users might even miss altogether.